wiki/People/en.md

```diff
@@ -0,0 +1,13 @@
+# People
+
+Members of the osu!community that mantian *osu!*
+Most of the usergroups give special in-game/forum colours and/or profile badges.
```

Comment:
change to `Most of these user groups are given forum name colours, in-game chat colours, and/or profile badges.`